src/main/java/seedu/toluist/model/Task.java

```diff
@@ -83,9 +89,11 @@ public boolean equals(Object other) {
                 && this.description.equals(((Task) other).description)) // state check
                 && this.priority.equals(((Task) other).priority)
                 && this.allTags.equals(((Task) other).allTags)
+                && Objects.equals(this.recurringFrequency, ((Task) other).recurringFrequency) // handles null
```

Comment:
but once u mark complete 1 then they become equal right?
"add cca from/today 4pm to/today 5pm repeat/weekly"
"add cca from/next saturday 4pm to/next saturday 5pm repeat/weekly"
"mark complete 1"